rfcs/select-component.md

```diff
@@ -177,17 +177,17 @@ export default () => {
   * `SelectComponentIcon: ?React.ComponentType` component for all icons appearing in Select component. It's provided `$type: ICON.loop | ICON.clearTag | ICON.clearAll | ICON.selected` to setup corresponding icon of Select component
   * `Tag: ?React.ComponentType` component for selected options Tags shown in Input for multiple mode selection
   * `SelectSpinner: ?React.ComponentType` component for dropdown spinner if async options are loading
-* `onChange: func(e: SyntheticInputEvent, params: Object)`:
+* `onChange: func({event, type, selectedOptions})`:
```

Comment:
I think this needs `option` too:
![image](https://user-images.githubusercontent.com/875591/46637167-c5e70600-cb0f-11e8-8d9a-3db6b3095207.png)

Reply:
thanks, I've totally missed that! I am not sure if I 100% understand that option though - is that the option the user currently hovers over / navigated to, but did not selected yet? if so, shouldn't we use the `pendingOption` name externally too? cc @blackswanny

Reply:
it's the one , which is Selected\Unselected. OnChange does work only when we change selections